build/pr_check_inner.sh

```diff
@@ -73,4 +73,5 @@ $KUBEBUILDER_ASSETS/kubectl apply -f build/skuttl-namespace.yaml
 $KUBEBUILDER_ASSETS/kubectl apply -f build/skuttl-perms.yaml
 
 IMG=$IMAGE_NAME:$IMAGE_TAG make deploy
-$KUBEBUILDER_ASSETS/kubectl kuttl test --config bundle/tests/scorecard/kuttl/kuttl-test.yaml --crd-dir config/crd/bases/ bundle/tests/scorecard/kuttl/
\ No newline at end of file
+
+source build/run_kuttl.sh
```

Comment:
is there a reason we `source` here and `bash` in other places?

Reply:
I did this because PATH is set a few lines above and I wanted to preserve that